python/grass/docs/conf.py

```diff
@@ -431,3 +432,14 @@
 
 # Where class documentation comes from (class or __init__ docstring).
 autoclass_content = "both"
+
+# sphinx-sitemap extension config
+# https://sphinx-sitemap.readthedocs.io/en/latest/advanced-configuration.html
+sitemap_filename = "sitemap.xml"
+html_baseurl = "https://grass.osgeo.org/"
+sitemap_url_scheme = "grass{version}manuals/libpython/{link}"
```

Comment:
```suggestion
sitemap_url_scheme = "grass{version}/manuals/libpython/{link}"
```

Reply:
It adds a / automatically.

Reply:
Well, it is indeed the documented behavior. I guess we are lucky that we have version at the end of directory name and not in the middle.